apps/authentication-service/src/middleware/auth0.middleware.ts

```diff
@@ -0,0 +1,10 @@
+import * as dotenv from "dotenv";
+import { auth } from "express-oauth2-jwt-bearer";
+
+dotenv.config();
+
+export const validateAccessToken = auth({
```

Comment:
Is this not being used?

Reply:
Yeah, not yet. I got tired of waiting for an approval, and let some of my next code sneak in.